app/helpers/welcome_helper.rb

```diff
@@ -41,4 +41,8 @@ def google_site_verification_meta_tag
       tag("meta", name: "google-site-verification", content: ENV["GOOGLE_SITE_VERIFICATION_ID"])
     end
   end
+
+  def analytics_valid?
```

Comment:
@radinreth please move this method to app/model/setting.rb as a helper is using in a purpose of UI rendering but not logical checking

Reply:
You're right , helpers used to support UI, if we moved to setting ( it's mean we moved it another layer(model))
but anyway, i will notify you when done